de_web_plugin.cpp

```diff
@@ -4286,7 +4282,8 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const deCONZ::
                     //         fpSwitch.outClusters.push_back(ci->id());
                     //     }
                     // }
-                    else if (node->nodeDescriptor().manufacturerCode() == VENDOR_JENNIC)
+                    else if (node->nodeDescriptor().manufacturerCode() == VENDOR_JENNIC ||
+                             modelId.contains(QLatin1String("86opcn01"))) // Aqara Opple prevent client clusters creation
                     {
                         // prevent creation of ZHASwitch, till supported
```

Comment:
Best put a comment that client clusters aren't used now fpSwitch is created for 0xfc00 cluster.

Reply:
Done!